### PR TITLE
Google tag manager

### LINKED ISSUE #711: Google Tag Manager Integration

### PR DESCRIPTION
**Pull request by ashishpachauriap**
close #711

### CODE REVIEW

**Review by meetmandeep — CHANGES_REQUESTED** [review]
@nikhilmandeeps Please resend with requested changes.

File: DesktopModules/Vanjaro/UXManager/Extensions/Menu/GoogleTagManager/Views/Setting/Setting.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
+﻿<div ng-controller="setting_setting" class="vj-ux-manager">
+    <uiengine provider="AngularBootstrap" identifier="setting_setting" class="uiengine-wrapper scrollbar">
+        <uilayout name="default">
+            <uielement name="ApplyTo" control="checkboxtoggle" label="false" attr-class="toggle_right btn_integration" container-attr-ng-show="ui.data.IsSuperUser.Options" ngmodel="ui.data.ApplyTo.Options"></uielement>
+        </uilayout>
+        <uilayout name="default" attr-ng-show="ui.data.ApplyTo.Options">
+            <uielement name="Host_Tag" control="textarea" attr-rows="20"></uielement>
+        </uilayout>
+        <uilayout name="default" attr-ng-show="!ui.data.ApplyTo.Options">
+            <uielement name="Site_Tag" control="textarea" attr-rows="20"></uielement>
```
Comment (nikhilmandeeps) [inline DesktopModules/Vanjaro/UXManager/Extensions/Menu/GoogleTagManager/Views/Setting/Setting.html]: please add head tag and body tag text area control and remove tag text area control

**Review by nikhilmandeeps — CHANGES_REQUESTED** [review]

File: DesktopModules/Vanjaro/UXManager/Extensions/Menu/GoogleTagManager/Views/Setting/Setting.html (hunk 1 above)
Comment (ashishpachauriap) [reply DesktopModules/Vanjaro/UXManager/Extensions/Menu/GoogleTagManager/Views/Setting/Setting.html]: done the required changes, Please review

File: Portals/_default/Skins/Vanjaro/Base.ascx.cs [hunk 2]
```diff
@@ -150,8 +152,16 @@ protected void Page_Load(object sender, EventArgs e)
...
+
+
+
+
+
+
+
                 ClientResourceManager.RegisterScript(Page, Page.ResolveUrl("~/Portals/_default/Skins/Vanjaro/Resources/js/skin.js"), 2, "DnnFormBottomProvider");
-                ClientResourceManager.RegisterScript(Page, Page.ResolveUrl("~/DesktopModules/Vanjaro/Common/Frameworks/Bootstrap/5.0.0/js/bootstrap.min.js"), 1, "DnnFormBottomProvider");
+                ClientResourceManager.RegisterScript(Page, Page.ResolveUrl("~/DesktopModules/Vanjaro/Common/Frameworks/Bootstrap/4.5.0/js/bootstrap.min.js"), 1, "DnnFormBottomProvider");
```
Comment (nikhilmandeeps) [inline Portals/_default/Skins/Vanjaro/Base.ascx.cs]: Please undo the line for bootstrap change the version 5.0.0 to 4.5.0

File: Portals/_default/Skins/Vanjaro/Base.ascx.cs [hunk 3]
```diff
@@ -369,8 +419,6 @@ private void AddModulePanes()
                 html.LoadHtml(sb.ToString());
                 CheckPermission(html);
                 InjectBlocks(page, html);
-                if (!PageManager.InjectEditor(PortalSettings))
```
Comment (nikhilmandeeps) [inline Portals/_default/Skins/Vanjaro/Base.ascx.cs]: Please merge upstream develop branch for change of this line

Comment (ashishpachauriap) [reply Portals/_default/Skins/Vanjaro/Base.ascx.cs]: Upstream develop branch has been merge

File: Portals/_default/Skins/Vanjaro/Base.ascx.cs (hunk 2 above)
Comment (ashishpachauriap) [reply Portals/_default/Skins/Vanjaro/Base.ascx.cs]: Merging upstream develop branch has resolved this

**Review by nikhilmandeeps — APPROVED** [review]

**Review by meetmandeep — APPROVED** [review]

**Pull request merged**